ansible/roles_ocp_workloads/ocp4_workload_5gran_deployments_lab/tasks/workload.yml

```diff
@@ -9,6 +9,20 @@
   register: result
   failed_when: result.rc != 0 or ("skipped on local" not in result.stdout and "deployed on local" not in result.stdout)
 
+- name: Ensure kubernetes manifests are downloaded
+  ansible.builtin.get_url:
+    url: "{{ item.url }}"
+    dest: "{{ item.destination }}"
+    mode: "{{ item.mode }}"
+  # yamllint disable rule:line-length
+  with_items:
+    - {url: "https://raw.githubusercontent.com/RHsyseng/5g-ran-deployments-on-ocp-lab/main/lab-materials/lab-env-data/hub-cluster/lvmcluster.yaml", destination: "/tmp/lvmcluster.yaml", mode: "0644"}
+    - {url: "https://raw.githubusercontent.com/RHsyseng/5g-ran-deployments-on-ocp-lab/main/lab-materials/lab-env-data/hub-cluster/argocd-patch.json", destination: "/tmp/argocd-openshift-gitops-patch.json", mode: "0644"}
+    - {url: "https://raw.githubusercontent.com/RHsyseng/5g-ran-deployments-on-ocp-lab/main/lab-materials/lab-env-data/hub-cluster/hub-operators-argoapps.yaml", destination: "/tmp/hub-operators-argoapps.yaml", mode: "0644"}
+    - {url: "https://raw.githubusercontent.com/RHsyseng/5g-ran-deployments-on-ocp-lab/main/lab-materials/lab-env-data/hub-cluster/sno1-argoapp.yaml", destination: "/tmp/sno1-argoapp.yaml", mode: "0644"}
+    - {url: "https://raw.githubusercontent.com/RHsyseng/5g-ran-deployments-on-ocp-lab/main/lab-materials/lab-env-data/hub-cluster/argocd-clusterrolebinding.yaml", destination: "/tmp/argocd-clusterrolebinding.yaml", mode: "0644"}
+  # yamllint enable rule:line-length
+
 - name: Wait for hub cluster to be deployed via kcli
   ansible.builtin.shell:
     cmd: oc login https://{{ lab_api_host }} -u admin -p admin --insecure-skip-tls-verify=true > /dev/null
```

Comment:
admin / admin?
Shouldn't we generate a strong password and pass it down as agnosticd_user_info?
see https://redhat-cop.github.io/agnosticd/#_environment_data

Reply:
This is for a hands-on lab where this cluster is not exposed anywhere. Is a isolated cluster, we could put a stronger password but since the lab content will be public, the password would be exposed anyway

Reply:
Added as additional commit